File: SofiForce.Models/Models/EntityModels/ClientPlanModel.cs
using System;

namespace Models
{
    public class ClientPlanModel
    {
        public long PlanId { get; set; }
        public int? ClientId { get; set; }
        public string ClientCode { get; set; }
        public DateTime? TargetDate { get; set; }
        public decimal? TargetValue { get; set; }
        public int? TargetVisit { get; set; }
        public int? TargetCall { get; set; }

    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/ClientPlanModel.ts
import { ClientModel } from "./clientModel";

export interface ClientPlanModel {
    planId: number;
    clientId: number | null;
    planYear: number | null;
    planMonth: number | null;
    targetValue: number | null;
    targetVisit: number | null;
    targetCall: number | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
    clientCode: string | null;
}